guides/MOCK.md

```diff
@@ -23,13 +23,11 @@ More options for the `meeshkan mock` command an be seen by running `meeshkan moc
 
 ## Callbacks
 
-A directory containing callbacks can be provided in callback_path argument. The default is:
-```bash
-meeshkan mock --callback_path ./callbacks
+To customize responses a directory containing callbacks can be provided in the `callback-path` argument (default: `./callbacks`).
+
+This directory can contain multiple Python scripts containing callbacks. A callback is a function decorated as 
+`tools.meeshkan_server.server.callbacks.callback`, each being mapped to an endpoint by a HTTP method, host and path.
```

Comment:
Do we even have the `tools` package anymore?

Reply:
Thanks, fixed that!